packages/babel-types/src/index.js

```diff
@@ -431,7 +424,6 @@ export function inherits(child: Object, parent: Object): Object {
   }
 
   t.inheritsComments(child, parent);
-  traverse.copyCache(parent, child);
```

Comment:
This is the one logic change, but removing it causes no test failures, and nothing obvious comes to mind that would be broken by this.